lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -104,7 +104,20 @@ check_content_in_yaml_editor:
   element:
     selector:
       xpath: //*[contains(@class,'react-monaco-editor-container')]//*[contains(text(),'<yaml_content>')]
-
+check_sidebar_item:
+  element:
+    selector:
+      xpath: //li[@class='co-resource-sidebar-item']//*[contains(text(),'<sidebar_item>')]
+click_on_resource_item:
+  element:
+    selector:
+      xpath: //a[@class='co-resource-item__resource-name' and @data-test-id='<resource_name>']
+    op: click
+wait_dropdown_loaded:
+  element:
+    selector:
+      xpath: //*[contains(@class, 'dropdown-menu--text-wrap')]
+    timeout: 20
```

Comment:
Mistakenly removed from #494, add them back

Reply:
This part LGTM